cmd/processManifest.go

```
@@ -12,20 +22,74 @@ var processManifestCmd = &cobra.Command{
 	Short: "Process a given manifest to meet the rhmi requirements.",
 	Long:  `Process a given manifest to meet the rhmi requirements.`,
 	Run: func(cmd *cobra.Command, args []string) {
-		fmt.Println("processImageManifests called")
+		//verify it's a manifest dir.
+		err := utils.VerifyManifestDirs(manifestDir)
+		if err != nil {
+			handleError(err)
+		}
+		//get current csv object and filepath
+		csv, filepath, err := utils.GetCurrentCSV(manifestDir)
+		if err != nil {
+			handleError(err)
+		}
+		if filepath == "" {
+			handleError(fmt.Errorf("No csv file found in the directory"))
+		}
```

Is filepath likely to be empty. If no csv file was found i think GetCurrentCSV should throw an error?

---

Yes, I even skipped testing that inside the GetSortedCSVs method because that's tested elsewhere. Will remove this in the morning. 👍